types/replicatedservice/v1/replicatedservice.py

```diff
@@ -41,9 +41,9 @@ def GenerateConfig(context):
       'properties': {
           'apiVersion': 'v1',
           'kind': 'Service',
-          'namespace': namespace,
           'metadata': {
               'name': service_name,
+              'namespace': namespace,
```

Comment:
Are you sure the namespace moves here? As I read the api, it's part of the path for creating the resource and not part of metadata?